BrightFutures/ExecutionContext.swift

```diff
@@ -50,9 +50,22 @@ public func toContext(queue: dispatch_queue_t) -> ExecutionContext {
     return Queue(queue: queue).context
 }
 
-typealias ThreadingModel = () -> ExecutionContext
+/// Defines BrightFutures' default threading behaviour, by default this is
+/// set to `StandardThreadingModel.AsynchronousOnCurrentThread`
+public var DefaultThreadingModel: StandardThreadingModel = .AsynchronousOnCurrentThread
```

Comment:
I understand your concerns, but hey, it's like "let's forbid sport cars, they are too dangerous".

Moreover, if it was possible to substitute threading model to a completely custom one through a "hack", frankly I would not bother too much. Still, I don't think it's a good idea to provide a limited functionality.

What I think as of the best practice, it's to provide a default behaviour, make it still customisable, leave a note in the docs, like "hey, it's dangerous, think twice, do you really need it?". This usually works perfectly and it will make it possible for more users to be satisfied from BrightFutures.

Mary Christmas guys and have a nice weekend.